READMEs.md

```diff
@@ -17,6 +18,14 @@ General GFM style pointers
 * Use sub README's where required i.e. if the repo has a "scripts" directory then it should have an accompanying README to describe what the scripts do, how to use them, etc.
 * Always link to sub README's from the main/top level README
 
+To help with support in service repos there should be subordinate READMEs linked to the main README as follows
```

Comment:
This reads as compulsory and some Repo's will not need these, should we rewrite it to use the words `where applicable` somewhere?